.gitmodules

```diff
@@ -2,6 +2,9 @@
 	path = backend/check_stac_metadata/geojson-spec
 	url = https://github.com/geojson/schema.git
 	branch = gh-pages
+[submodule "stac"]
```

Comment:
FYI, this name is maybe a bit weird. I originally called it `linz-stac`, but using the original name meant that I could use `LATEST_LINZ_SCHEMA_DIRECTORY` all over the place.